src/GitHub.Exports/Helpers/SimpleRepositoryModelExtensions.cs

```diff
@@ -0,0 +1,25 @@
+﻿using GitHub.Models;
+using System;
+using System.Linq;
+using System.IO;
+
+namespace GitHub.Extensions
+{
+    using VisualStudio;
+
+    public static class SimpleRepositoryModelExtensions
+    {
+        public static bool HasCommits(this ISimpleRepositoryModel repository)
+        {
+            var repo = Services.GetRepoFromPath(repository.LocalPath);
+            return repo?.Commits.Count() > 0;
+        }
+        public static bool HasContent(this ISimpleRepositoryModel repository)
+        {
+            var dir = new DirectoryInfo(repository.LocalPath);
+            return dir.EnumerateFileSystemInfos("*", SearchOption.TopDirectoryOnly)
+                      .Any(x => ((x.Attributes.HasFlag(FileAttributes.Directory) || x.Attributes.HasFlag(FileAttributes.Normal)) &&
+                                !x.Name.StartsWith(".", StringComparison.Ordinal) && !x.Name.StartsWith("readme", StringComparison.OrdinalIgnoreCase)));
```

Comment:
By content I mean a solution or project file that VS can open, mainly. I don't want to waste time scanning for explicit files, this is just a simple naive check to see if the repo has more than just the default content of .git files and the readme, which is a decent clue to use in deciding whether to show the create new or the open project dialog.

It could be that someone has created a repo with no subdirectories and a project and solution files are both called Readme.csproj/sln - that would totally fool this check :smile:

Otherwise, if the user is cloning a repo in VS, it's either because they want to open an existing project (in which case there will be something more in there besides the readme) or create a new solution (so the repo will be initialized but only with our default stuff). Either case should be decently covered by this best guess check.